ci-operator/step-registry/additional-enabled-capabilities/additional-enabled-capabilities-ref.yaml

```diff
@@ -0,0 +1,24 @@
+ref:
+  as: additional-enabled-capabilities
```

Comment:
I suggest  to put it under `ipi/conf` folder, means it's an extra configuration for `install-config.yaml`, wdyt?

Reply:
good suggestion, will update.